crates/ruff_python_resolver/src/lib.rs

```diff
@@ -129,6 +128,14 @@ mod tests {
         env_logger::builder().is_test(true).try_init().ok();
     }
 
+    macro_rules! assert_debug_snapshot_normalize_paths {
+        ($value: ident) => {
```

Comment:
I think you want to use an extra pair of `{` to ensure `value` doesn't leak into the outer scope
```suggestion
        ($value: ident) => {{
```